configs/linux.config

```diff
@@ -734,40 +830,53 @@ CONFIG_SERIAL_8250_DEPRECATED_OPTIONS=y
 # CONFIG_SERIAL_8250_16550A_VARIANTS is not set
 # CONFIG_SERIAL_8250_FINTEK is not set
 CONFIG_SERIAL_8250_CONSOLE=y
+CONFIG_SERIAL_8250_PCI=y
```

Comment:
Is serial 8250 PCI necessary?

Reply:
You should check the Linux kernel configurations and enable the required options for virtio-pci.